master/buildbot/newsfragments/python3-irc-bytes.bugfix

```diff
@@ -0,0 +1,3 @@
+:py:class:`buildbot.reporters.irc.IrcStatusBot` no longer encodes messages
```

Comment:
I believe it's the best to follow one-sentence-per-line in every bit of the documentation

Reply:
ReST is spelled like that

Reply:
I don't understand these comments:

> I believe it's the best to follow one-sentence-per-line in every bit of the documentation

Sphinx takes care of line wrapping for output, and input is much more readable with sane linebreaks.

> ReST is spelled like that

I don't see where this should apply.

Reply:
(I cannot see now why ReST comment was made, sorry)

As for one-sentence-per-line, it's easier during the reviews.  I believe some insights could be seen in http://rhodesmill.org/brandon/2012/one-sentence-per-line/

Reply:
Thanks for an interesting read; I'll try to keep that in mind in future.  However I don't think it makes much sense here as this is a short-lived file and the simpler diffs argument doesn't really apply when it's all additions no matter where the linebreaks are.

If flaky CI forces me to push again, I will consider additional linebreaks here as a simple change to allow a fresh push :)